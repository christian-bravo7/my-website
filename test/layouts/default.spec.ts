import { createLocalVue, shallowMount } from '@vue/test-utils';
import Vuex from 'vuex';

import Default from '@/layouts/default.vue';

const localVue = createLocalVue();
localVue.use(Vuex);

describe('default layout', () => {
  let getters: any;
  let store: any;

  beforeEach(() => {
    getters = {
      'guest/isAppInitialized': jest.fn(() => false),
    };
    store = new Vuex.Store({
      getters,
    });
  });

  it('should render correclty', () => {
    const wrapper = shallowMount(Default, {
      store,
      localVue,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
