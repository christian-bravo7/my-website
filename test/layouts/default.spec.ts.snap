// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default layout should render correclty 1`] = `
<main class="bg-gray-50 dark:bg-dark-theme-800">
  <appfallbackload-stub></appfallbackload-stub>
  <appsidebarmenu-stub haslanguagepicker="true" style="display: none;"></appsidebarmenu-stub> <span><appnavigationbar-stub></appnavigationbar-stub></span>
  <header>
    <bannerbackgroundfigures-stub></bannerbackgroundfigures-stub> <span name="page-banner" slim="" class="min-h-screen"></span>
  </header> <span></span>
  <appfooter-stub></appfooter-stub>
</main>
`;
